Networking/Networking/Model/Product/ProductImage.swift

```diff
@@ -37,7 +37,14 @@ public struct ProductImage: Codable, Equatable, GeneratedCopiable, GeneratedFake
         let dateModified = try container.decodeIfPresent(Date.self, forKey: .dateModified)
         let src = try container.decode(String.self, forKey: .src)
         let name = try container.decodeIfPresent(String.self, forKey: .name)
-        let alt = try container.decodeIfPresent(String.self, forKey: .alt)
+        let alt: String? = {
+            do {
+                return try container.decodeIfPresent(String.self, forKey: .alt)
+            } catch {
+                DDLogError("⛔️ Error parsing `alt` for `ProductImage`: \(error)")
```

Comment:
Same as before, would be useful to pass the `imageID`? 
```
DDLogError("⛔️ Error parsing `alt` for `ProductImage \(imageID)`: \(error)")
```

Reply:
Also updated in 2bed57dd321e45b2c8b08fc6485cd21c87497071 👍